src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -929,6 +929,32 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "ExposureSubmissionWarnOthers_continueButton" = "Einverstanden";
 
+/* Exposure Submission Warn Others Test Type Selection */
+
+"ExposureSubmission_TestTypeSelection_title" = "Art des Tests";
+
+"ExposureSubmission_TestTypeSelection_body" = "Bitte wählen Sie die Art des Tests aus, auf dessen Grundlage Sie warnen.";
+
+"ExposureSubmission_TestTypeSelection_optionSRSSelfTest_title" = "Antigen-Schnelltest nicht in der App registriert";
```

Comment:
Can we have SRS before option and also for others on the bottom?
```suggestion
"ExposureSubmission_TestTypeSelection_SRSoptionSelfTest_title" = "Antigen-Schnelltest nicht in der App registriert";
```

Reply:
yes, good suggestion, changed everywhere